### ISSUE TITLE
SC2020 false warning for tr character classes

### ISSUE THREAD

**Issue by DasFaultier** [koalaman/shellcheck#2457]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2020
- My shellcheck version (`shellcheck --version` or "online"): 0.7.0 (as included in Linux Mint 20.3 and Ubuntu Focal)
- [X] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [X] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [X] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/usr/bin/env bash
tr ':upper' ':lower'

```

#### Here's what shellcheck currently says:

SC2020: tr replaces sets of chars, not words (mentioned due to duplicates).

#### Here's what I wanted or expected to see:

The tr call from the example above will relpace all uppercase letters to lowercase letters using character classes. Shellcheck should not give a warning for translating character classes.

**Comment by brother** [comment]
Shellcheck is correct in complaining about that syntax. The character classes should be in square brackets and colons like so:
```
#!/usr/bin/env bash
tr '[:upper:]' '[:lower:]'
```

Shellcheck does not complain about this syntax.

**Comment by DasFaultier** [comment]
Sorry, you're right, I must've misunderstood the manpage. If you think that the error message should/could convey more clearly what the problem is, then feel free to keep the bug open, but as the original issue has dissolved, I don't think this needs fixing no matter what.
Thans for looking into this.
